pkg/controllers/scheduler/schedulingunit.go

```diff
@@ -152,6 +154,20 @@ func schedulingUnitForFedObject(
 		schedulingUnit.MaxClusters = maxClustersOverride
 	}
 
+	gvk := schedulingUnit.GroupVersion.WithKind(schedulingUnit.Kind)
+	podSpec, err := fedpodutil.GetResourcePodSpec(fedObject, gvk)
+	// if we cannot recognize the type of resourceObj,
+	// do not return so that it will not block the entire scheduling process.
+	if err == nil {
+		resourceQuest := *framework.NewResource(resourceutil.GetPodResourceRequests(podSpec))
```

Comment:
We obtain the gpu resources from requests, what if users specify GPU limits without specifying requests

Reply:
https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/
![image](https://github.com/kubewharf/kubeadmiral/assets/74236054/9ee7ba8d-ecae-4ae5-a598-d5dde0b33f8b)

Reply:
![img_v2_dff14454-79da-44fb-8fe0-52f78ccfa04g](https://github.com/kubewharf/kubeadmiral/assets/26862112/b77bdaba-00c5-4016-932c-f36f4dc44ff0)

Reply:
I made a mistake here. In the actual test, the Pod's spec will automatically fill in the request value, but the Deployment and so on will not. Since we get the request from the deployment's spec, we still need to deal with this special situation. @mrlihanbo @manmanhappy